examples/test-app/steps/Country/template.html

```diff
@@ -8,19 +8,19 @@
 {% set phase="ALPHA" %}
 {% set feedbackLink="https://github.com/hmcts/one-per-page/issues/new" %}
 {% set title %}
-  Where do you live?
+  {{ content.title | safe }}
 {% endset %}
 
 {% block fields %}
 
   {% call formSection() %}
-    {{ selectionButtons(fields.get('country'), "Where do you live?",
-      hint = 'Some advice about the question',
+    {{ selectionButtons(fields.get('country'), fields.get('country').content.title,
+      hint = fields.get('country').content.hint,
       options = [
-        { label: "Northern Ireland", value: "northern-ireland" },
-        { label: "Scotland", value: "scotland" },
-        { label: "England", value: "england" },
-        { label: "Wales", value: "wales" }
+        { label: fields.get('country').content.northanIreland, value: "northern-ireland" },
+        { label: fields.get('country').content.scotland, value: "scotland" },
+        { label: fields.get('country').content.england, value: "england" },
+        { label: fields.get('country').content.wales, value: "wales" }
```

Comment:
I realise this was in a previous PR but it would be preferable, if possible, to reference the fields without the get() function. For example: `fields.country.content.england` not sure if that's possible but is less verbose. Something for a later date.

Reply:
i tried to do this, but couldnt get there without much bigger changes, will put in back log as future suggestion